includes/tracks/wc-site-tracking.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Nosara Tracks for Woo
+ */
+
+require_once dirname( __FILE__ ) . '/class-wc-tracks.php';
+require_once dirname( __FILE__ ) . '/class-wc-tracks-event.php';
+require_once dirname( __FILE__ ) . '/class-wc-tracks-client.php';
+
+/**
+ * Send a Tracks event when a product is updated.
+ *
+ * @param int   $product_id Product id.
+ * @param array $post WordPress post.
+ */
+function woocommerce_tracks_product_updated( $product_id, $post ) {
```

Comment:
I assume this file is a sample we'll update and review later.

Reply:
This is an event that we'll be tracking, so its best to review now so that this event doesn't go without some scrutiny.

Reply:
Ok; well in that case:

- This would probably make sense as a class, with a method to add all the hooks and a method per event.
- Would need to know more about intention of the events to see if the used hook is a good fit. For example the current event runs on edit_post which is fine for WP Admin, but won't track product saves outside of admin e.g. edits by plugins

Reply:
>Would need to know more about intention of the events to see if the used hook is a good fit. For example the current event runs on edit_post which is fine for WP Admin, but won't track product saves outside of admin e.g. edits by plugins.

Great question. I think the nature of these events is to make inferences about usage of the Woo interface only. So modification of products by plugins don't need to be tracked. @warrendholmes is that correct?